example/apache-ignite/src/main/scala/org/finos/vuu/example/ignite/IgniteVuuMain.scala

```diff
@@ -63,12 +63,8 @@ object IgniteVuuMain extends App with StrictLogging {
 
   val config = VuuServerConfig(
     VuuHttp2ServerOptions()
-      //only specify webroot if we want to load the source locally, we'll load it from the jar
-      //otherwise
-      .withWebRoot(webRoot)
+      .withWebRoot(AbsolutePathWebRoot(webRoot, directoryListings = true))
```

Comment:
I know its always been this way, but do we actually want directoryListings true

I assume this is what it sounds like and allows user to enter  
https://localhost:8443 
and see directory listing for public .

Alternative is we expect them to use one of

- https://localhost:8443/index.html
- https://localhost:8443/login.html
- https://localhost:8443/demo.html